### DISCUSSION TITLE
Reveal.js template for Replit.com

### DISCUSSION THREAD

**Discussion by sflanker** [hakimel/reveal.js#3364] [Show and tell]
First I just wanted to say this project is excellent and I'm really grateful that it exists.

I use Replit.com for a lot of experimentation and web development and I wanted to be able to create Reveal.js presentations there so I created this template: https://replit.com/@KumuPaul/Revealjs?v=1

It includes the boiler plate of instantiating Reveal.js with some good default configuration, hiding un-styled markup until Reveal.js is initialized, and provides markup for some basic example slides.

It uses Vite.js to provide a hot-reloading development server for when you're actively editing, and can also build your presentation as static HTML/js for deployment as static content.

Enjoy!

**Comment by Martinomagnifico** [comment]
Hi Paul, that looks very promising. Thanks!
